src/components/erc721/collectibles/collectibles_list.tsx

```diff
@@ -101,16 +104,16 @@ const ButtonStyled = styled(Button)`
     }
 `;
 
-export class CollectiblesList extends React.Component<Props, State> {
-    public state = {
-        sortType: CollectibleSortType.NewestAdded,
-        filterType: CollectibleFilterType.ShowAll,
+export class CollectiblesList extends React.Component<Props, {}> {
+    public componentWillUnmount = () => {
```

Comment:
This removes the `sort` and `filter` search params from the URL when the components is unmounted.